### PR TITLE
Add the new command 'Attach Source' to the readme

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

### CODE REVIEW

**Comment by fbricon** [comment]
also please change the commit message, replace wiki with readme

File: README.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ The following commands are available:
 *New in 0.33.0:*
```
Comment (fbricon) [inline README.md]: please remove the line above, as I commented before, so that command is moved with the existing ones

Comment (testforstephen) [reply README.md]: Got it. thanks.

**Comment by fbricon** [comment]
Thanks @testforstephen

**Pull request merged**